global.json

```diff
@@ -1,6 +1,11 @@
 {
   "tools": {
```

Comment:
What is the purpose of the change to global.json?  Is that expected in this review?

Reply:
Oh I missed that comment.